lib/ceedling/setupinator.rb

```diff
@@ -16,14 +17,16 @@ def load_project_files
 
   def do_setup(config_hash)
     @config_hash = config_hash
+    @defaults_hash = {}
```

Comment:
Does this need to be an instance variable? If not then you could remove the `@` (and no need to initialise it in `setup`), or alternatively you could consider adding it to the `attr_reader` line above so it can be accessed from outside similar to `config_hash`.